src/main/java/com/shyb/evcharging/EvChargingApplication.java

```diff
@@ -1,12 +1,13 @@
 package com.shyb.evcharging;
 
 
+import com.shyb.evcharging.card.config.MyBatisCardConfig;
 import com.shyb.evcharging.user.config.MyBatisUserConfig;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Import;
 
-@Import(MyBatisUserConfig.class)
+@Import({MyBatisUserConfig.class, MyBatisCardConfig.class})
```

Comment:
만약 Import를 의도하신거라면 Config의 Configuration 어노테이션을 제거하면 좋을 것 같아요.

그리고 Config가 늘어남에 따라 계속 Import의 개수가 늘어나는 구조인데 아래와 같은 형태로 사용하면 어떨까요?

```suggestion
@Import(MyBatisConfig.class)
```

MybatisConfig에 각 도메인(User, Card) 별 설정을 넣어두시면 이쪽은 변경될 필요 없이 추가되는 설정을 로드할 수 있을것 같아요.